container.yaml

```diff
@@ -0,0 +1,5 @@
+compose:
+  pulp_repos: true
+platforms:
+  only:
+  - x86_64
```

Comment:
Are you building only for x86_64?

Reply:
Yes, only for x86_64 for now (ppc64, ppc64le) wasn't investigated so far AFAICT>